src/pre_commit_vauxoo/cli.py

```diff
@@ -13,27 +13,135 @@
 
   Also see (1) from http://click.pocoo.org/5/setuptools/#setuptools-integration
 """
+
+
+import contextlib
 import os
 
 import click
 
 from pre_commit_vauxoo import pre_commit_vauxoo
 
 
-def strcsv2tuple(strcsv, is_path=False):
+def source_variables():
+    # Overwrite os.environ with variables.sh file only if it was not already defined
+    repo_dirname = pre_commit_vauxoo.get_repo()
+    envdict = pre_commit_vauxoo.envfile2envdict(repo_dirname)
+    for var, value in envdict.items():
+        if var in os.environ:
+            continue
+        os.environ[var] = value
```

Comment:
Updating using dict comprehension would be faster, as there will be only one update instead of one per var